File: Deberes/EXAMEN_IIBIMESTRE/app/src/main/java/com/example/examen_iibimestre/Equipo.kt
package com.example.examen_iibimestre

class Equipo(
    public var nombre:String,
    var liga:String,
    var fechaCreacion:String,
    var numeroCopasInternacionales:Int,
    var campeonActual:Boolean,
    var id:Int

) {
    override fun toString(): String {

        return this.nombre
    }
}

File: 05_backend/backend-movil/api/models/Equipo.js
/**
 * Equipo.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

    nombre: {
      type: 'string',
      required: true,
      minLength: 3,
      maxLength: 60,
    },

    liga: {
      type: 'string',
      required: true,
      minLength: 3,
      maxLength: 60,
    },

    fechaCreacion: {
      type: 'string',
      required: true,
    },

    numeroCopasInternacionales: {
      type: 'number',
      required: true,
    },

    campeonActual: {
      type: 'boolean',
      required: true,
    },

    jugadorEquipo:{
      collection:'jugador',
      via:'equipoFutbolId'
    }

  },

};

